dcplib/security/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""
+This library contains utilities to use secure features of the DCP.
+
+"""
+
+from typing import List
+
+from .dcp_service_account import DCPServiceAccountManager
+from ..errors import SecurityException
+
+security_config_not_set_error = "security configuration not set for {}. Use Config.setup() to set."
+
+
+class Config:
+    _openid_provider = "humancellatlas.auth0.com"
+    _auth_provider = "https://auth.data.humancellatlas.org"
+    _trusted_google_projects = None
+
+    @staticmethod
+    def setup(openid_provider: str = None, auth_url: str = None, trusted_google_projects: List[str] = None):
```

Comment:
Consider making these keyword-only args.

Reply:
That's a new feature to me. I'll give it a try.